pkg/kubectl/cmd/logs.go

```diff
@@ -94,9 +100,7 @@ func NewCmdLogs(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, out, cmd, args))
-			if err := o.Validate(); err != nil {
-				cmdutil.CheckErr(cmdutil.UsageError(cmd, err.Error()))
-			}
+			cmdutil.CheckErr(o.Validate())
```

Comment:
why remove this ? if a user provided an invalid logs options, shouldn't we prompt this ?
```console
unexpected logs options object 
See 'logs -h' for help and examples.
```